### PR TITLE
Add declaration questions for DOS

### PR DESCRIPTION
**Pull request by quis**
This is a dump of all the questions in this document:
https://docs.google.com/document/d/1pxM52rDLL4HTWKXJHZf2t_TkD8dj9sCE2MjcGYDZS9Y/edit?ts=561ba886

This is so we can start iterating on them in a sensible way and track the changes in Git. It's not suggested that the wording or format of the questions is 'right' yet.

### CODE REVIEW

**Comment by TheDoubleK** [comment]
Good effort :+1: 

Although it's going to make @pcraig3's current story a lot bigger!

File: frameworks/digital-outcomes-and-specialists/questions/declaration/davidKershaw.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+question: >
```
Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/declaration/davidKershaw.yml]: I think we need a different name for this one.

Comment (quis) [inline frameworks/digital-outcomes-and-specialists/questions/declaration/davidKershaw.yml]: The name is just temporary—it's going to be split into multiple questions.

**Comment by quis** [comment]
Going to close this until the questions have had some attention from @superroz on this branch

**Pull request closed without merging**